### PR TITLE
Issue #SB-6729 feat: Course notifications bugs fixes

### PR DESCRIPTION
**Pull request by iostream04**

### CODE REVIEW

File: actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchNotificationActor.java [hunk 1]
```diff
@@ -43,6 +44,11 @@
   private DecryptionService decryptionService =
       org.sunbird.common.models.util.datasecurity.impl.ServiceFactory.getDecryptionServiceInstance(
           null);
+  private static String courseBatchNotificationSignature =
```
Comment (bvinayakumar) [inline actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchNotificationActor.java]: Why JsonKey.SUNBIRD_COURSE_BATCH_NOTIFICATION_SIGNATURE is required? How are the existing email template signatures working without this? Where is the default value defined for this variable?

Comment (iostream04) [reply actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchNotificationActor.java]: Existing email template are using ORG name . Product Manager want it configurable. 
Default value is sunbird, it is added to env variable tracking google doc. and also informed in devops channel

Comment (bvinayakumar) [reply actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchNotificationActor.java]: Is this configurable per tenant?

Comment (iostream04) [reply actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchNotificationActor.java]: Yes

**Pull request merged**